workflow_creation/utilities_auto_gui.md

```diff
@@ -52,30 +97,22 @@ The _app_ directory contains a configuration file for the web site layout parame
 </pageConfiguration>
 ```
 
-#### 2.3 Excel-Sheet for GUI Elements:
-The _app_ directory also contains an excel sheet, which lists all gui elements. It allows to specify *element name*, *element type*, *unit of measurement*, *default value*, *min/max values*, *checked* state and the name of the *output parameter* for the workflow manager. The min/max parameters can be used to prevent users from specifying malicious parameters but are not mandatory. The checked state option is only used for checkable elements, i.e., checkboxes and radiobuttons. The unit of measurement option can be used to present the unit, e.g., milimeters, of the parameter to the user. Note that the unit of measurement option is mandatory for radiobuttons, since the application assigns radiobuttons of the same unit to the same group. For an complete example of a valid excel stylesheet checkout the *GUIAppParameters.xlsx* file in the *app* directory.
-
-Currently the application supports the following data types:
-+ **title**: The title of the web page. This shall be set first.
-+ **heading**: Any heading to inform users about the semantics of GUI elements
-+ **integer**: Integer input field
-+ **decimal**: Float input field
-+ **string**: String input field
-+ **3 decimals**: Three contiguous decimal input fields
-+ **6 decimals**: Six contiguous decimal input fields
-+ **radiobutton**: A single radiobutton. To form groups of radiobuttons specify same unit of measurement. Set the value of the *checked* field to *YES* for an initially checked radiobutton.
-+ **checkbox**: A single checkbox. Set the value of the *checked* field to *YES* for an initially checked checkbox.
-
-
 3 Deployment
 ---
-Before service deployment ensure that docker is installed and the service configuration is correct. The deployment of the application just amounts to building and starting the docker container. From the directory containing this README file execute the following:
+This section describes the local deployment of the _GenericGUIApplication_.
+Note that this description is only valid for [local deployment](../tutorials/workflows/basics\_service\_registration.md).

```

Comment:
Almost the right link. ;-)
I mean this one: `../service_implementation/deployment_automated.md`

Also, please split the sentences, it doesn't make sense currently. For example:
"Note that this description is only valid for local deployment. For deployment on the CloudFlow platform, see the [deployment manual](../service_implementation/deployment_automated.md)."

Reply:
I also deployed a demo version of the service and made a demo workflow from it. I suggest to further add the following here:

"A demo version of this service is deployed under the URI `http://demo/apps/startInputGUI.owl#startInputGUI_Service`. Furthermore, a demo workflow using this service is available under the name `Demo_InputGUI` (URI `http://demo/workflow/Demo_InputGUI.owl#Demo_InputGUI`)."

Reply:
Alright, I just committed the changes you proposed.